### PR TITLE
Group apis

### PR DESCRIPTION
**Pull request by AryanM1223**

### CODE REVIEW

File: controllers/groupController.js [hunk 1]
```diff
@@ -239,88 +232,85 @@ exports.removeUser = async (req, res) => {
...
-      isOpen: isOpen,
-      admin: admin,
-      members: list,
-      karma: karma,
+
+    // Create the group
+    const group = await Group.create({
+      name,
+      displayname,
+      icon: icon || 'default_icon.png',
```
Comment (BharatShukla30) [inline controllers/groupController.js:279]: what is this? what is default_icon.jpg

File: controllers/groupController.js [hunk 2]
```diff
@@ -239,88 +232,85 @@ exports.removeUser = async (req, res) => {
...
+      coordinates = user.current_coordinates.coordinates;
     }
-    let code = otpgenerator();
+
+    const location = {
+      type: "Point",
+      coordinates: coordinates
+    };
+
+    const code = otpgenerator();
```
Comment (BharatShukla30) [inline controllers/groupController.js:268]: why?

Comment (AryanM1223) [reply controllers/groupController.js:268]: to generate groupId

File: controllers/groupController.js (hunk 1 above)
Comment (AryanM1223) [reply controllers/groupController.js:279]: My mistake> will remove it

File: controllers/groupController.js [hunk 3]
```diff
@@ -239,88 +232,85 @@ exports.removeUser = async (req, res) => {
...
     }
-    group = await Group.create({
-      name: name,
-      displayname: displayname,
+
+    // Create the group
+    const group = await Group.create({
+      name,
+      displayname,
       icon: icon,
```
Comment (BharatShukla30) [inline controllers/groupController.js:279]: there should be some default icon. We are not taking icon from the user. Use some default color generation stuff here

**Review by BharatShukla30 — APPROVED** [review]

**Pull request merged**